src/Cache/_ScsDataClient.php

```diff
@@ -114,25 +114,43 @@ public function set(string $cacheName, string $key, string $value, int $ttlSecon
             $setRequest->setCacheBody($value);
             $setRequest->setTtlMilliseconds($ttlMillis);
             $call = $this->grpcManager->client->Set(
-                $setRequest, ["cache"=>[$cacheName]], ["timeout"=>$this->deadline_seconds * self::$TIMEOUT_MULTIPLIER]
+                $setRequest, ["cache" => [$cacheName]], ["timeout" => $this->deadline_seconds * self::$TIMEOUT_MULTIPLIER]
             );
-            $response  = $this->processCall($call);
+            $response = $this->processCall($call);
         } catch (SdkError $e) {
             return new CacheSetResponseError($e);
-        } catch (\Exception $e){
+        } catch (Exception $e) {
             return new CacheSetResponseError(new UnknownError($e->getMessage()));
         }
         return new CacheSetResponseSuccess($response, $key, $value);
     }
 
-    public function get(string $cacheName, string $key) : CacheGetResponse
+    private function ttlToMillis(?int $ttl = null): int
+    {
+        if (!$ttl) {
+            $ttl = $this->defaultTtlSeconds;
+        }
+        validateTtl($ttl);
+        return $ttl * 1000;
+    }
+
+    private function processCall(UnaryCall $call): mixed
+    {
+        [$response, $status] = $call->wait();
+        if ($status->code !== 0) {
+            throw _ErrorConverter::convert($status->code, $status->details, $call->getMetadata());
+        }
+        return $response;
+    }
```

Comment:
Can you move these definitions up so they live between `__construct()` and `set()` please?